prac_05/word_occurences.py

```diff
@@ -8,3 +8,56 @@
 - Align the numbers so they are in one nice column (see prac for notes)
 """
 
+from operator import itemgetter
+
+
+def main():
+    # requests phrase from user
+    phrase = input("enter a phrase:")
+    new_dictionary = convert_to_dict(phrase)
+    longest_word = get_longest_word(new_dictionary)
```

Comment:
Have a go at using a list comprehension to get the longest word in one line :)

Reply:
once i get through my assignment i'll revisit this one :)